app/models/location.py

```diff
@@ -0,0 +1,26 @@
+from .. import db
+
+
+class Zipcode(db.Model):
+    __tablename__ = 'zip_codes'
+    id = db.Column(db.Integer, primary_key=True)
+    zip_code = db.Column(db.String(10), unique=True)
```

Comment:
I'm all for extensibility, I just can't really think of a situation where we would benefit from a zip+4 and we definitely increase the complexity by doing so. I guess we could just store whatever the original input is even if we never use it.